docs/excel/excel-add-ins-events.md

```diff
@@ -133,27 +133,27 @@ function remove() {
 > [!NOTE]
 > This feature is currently available only in public preview (beta). To use it, you must reference the beta library of the Office.js CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
 
-Events are turned on and off at the [runtime](https://docs.microsoft.com/en-us/javascript/api/excel/excel.runtime?view=office-js) level. 
+Some scenarios benefit from the performance boost given by turning off events. 
+Your app might never need to receive events, or it could ignore events while performing batch-edits of multiple entities. 
```

Comment:
Maybe start this sentence with "For example, " ?  i.e.,  "For example, your app might..."